src/main/kotlin/sliv/tool/parsers/lines_parsers/ILinesParser.kt

```diff
@@ -0,0 +1,7 @@
+package sliv.tool.parsers.lines_parsers
+
+import sliv.tool.data_structures.landmarks.LineLandmark
+
+interface ILinesParser {
+    fun parse(filePath: String): List<LineLandmark>
```

Comment:
can we make single Interface parametrized by landmark type parameter as generic?